samples/authorization/csm_authorization_proxy_server_v190.yaml

```diff
@@ -9,16 +9,16 @@ spec:
     - name: authorization-proxy-server
       # enable: Enable/Disable csm-authorization
       enabled: true
-      configVersion: v1.6.0
+      configVersion: v1.9.0
```

Comment:
v1.9.0 manifests should be created from v1.8.0 manifests rather than v1.6.0 manifests. There are many differences between Authorization v1.6.0 and v1.8.0.